website/project/presentation.rst

```diff
@@ -37,37 +22,58 @@ non-trivial implementation details (like the use of external libraries).
+
+A good way
+to think about this section is to imagine that it will be read
+by students in following years who will continue working on
+your feature. When writing this part of the report, ask yourself what information
+would be useful to future students, and assume that they have
+the technical skills to read through the code and fill in
+all the lower-level details of your work (but that they
+need a good sense of the "big picture" before doing so).
+
+For example, if you added a new data structure called
+``foo_t``, it would be appropriate to mention that you
+did so, and to describe the purpose of that data structure,
+as well as any dependencies related to that data structure.
+On the other hand, you would not include the full declaration
+of that data structure, nor would you describe each individual
+field. Instead, the goal is to convey that ``foo_t`` is a data
+structure relevant to your feature, and that anyone working
+on that feature in the future will need to familiarize themselves
+in it (and the assumption is that those students will just look
+at the code to learn more about the specifics of that data structure).
+
+Along those lines, you should not need to include any code
+in this section of the report, although it can be
 appropriate to show very short examples (3-4 lines) of how your code
 might be called from other components.
 
-Demo
-~~~~
+There is no specific length requirement for this section, but we
+expect any sufficiently-detailed section to be at least 2-3 pages long.
 
-You must run a demo of your work meeting the following requirements:
+Next Steps
+~~~~~~~~~~
 
--  You must show a program in execution.
--  The code you run *must* be in the ``dev`` branch of the repository.
-   You cannot demo code that is still in one of your branches and has
-   not gone through a code review.
+Again, imagine that this report will be read by students who will
+continue your work in following years. Describe at four
+tasks that those students could work on to continue the work
+you did this quarter. These can include the following
 
-We recommend that you think carefully about what you want to show
-in your demo, and that you plan ahead of time the exact steps you
-will take during the demo (as opposed to doing a more improvised demo).
+- Tasks that you started working on this quarter, but were unable
+  to finish.
+- Improvements or fixes you identified this quarter, but did
+  not work on.
+- General interesting directions that the feature could go in.
 
-Ideally, you should run the main ``chiventure`` executable, in a way
-that highlights the work you’ve done. However, if your feature is not
-yet accessible from the main executable, you are allowed to create an
-example program inside the ``examples/`` directory of your feature.
+Please note that you can complete this section of the report
+by simply pointing to specific GitHub issues (particularly Backlog issues)
+that future students may want to focus on.
```

Comment:
Just to clarify, would it be ok if students submit this entire section just as "Next year, students can look at issues #123, #456, #789, and #101" or would we want more than that? If this is what we're looking for, then it might be good to specify in the scoring section that the issues are sufficiently detailed and documented.

Reply:
I've tweaked that paragraph to say that they have to include the issue number, issue title, and a 2-3 sentence summary of the issue.